src/app/core/guards/myfiles.guard.ts

```diff
@@ -0,0 +1,31 @@
+/* @format */
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import {
+  ActivatedRouteSnapshot,
+  CanActivate,
+  RouterStateSnapshot,
+  UrlTree,
+} from '@angular/router';
+import { Observable } from 'rxjs';
+
+/**
+ * Redirects any nested routes from /app/myfiles/** to /app/private/**
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class MyfilesGuard implements CanActivate {
```

Comment:
Why isn't this `MyFilesGuard`?

Reply:
This was just because of the filename I picked with the Angular CLI. I used "myfiles" to match what the URL looks like.